src/pager.rs

```diff
@@ -40,16 +40,24 @@ impl PagerKind {
     fn from_bin(bin: &str) -> PagerKind {
         use std::path::Path;
 
-        match Path::new(bin)
-            .file_stem()
-            .map(|s| s.to_string_lossy())
-            .as_deref()
-        {
+        let file_stem = Path::new(bin).file_stem();
+
+        match file_stem.map(|s| s.to_string_lossy()).as_deref() {
             Some("bat") => PagerKind::Bat,
             Some("less") => PagerKind::Less,
             Some("more") => PagerKind::More,
             Some("most") => PagerKind::Most,
-            _ => PagerKind::Unknown,
+            _ => {
+                if env::args_os()
```

Comment:
Sorry to nit-pick but I think this deserves its own variable with a clarifying comment. Something like:

```rust
// The bin name we run under. Normally `bat`, but can also be `batcat` for compatibility reasons.
let our_bin_name = env::args_os().next().
```

Reply:
Added it.